.github/workflows/build.yml

```diff
@@ -27,14 +27,14 @@ jobs:
       - name: Get yarn cache directory path
         if: steps.cache.outputs.cache-hit != 'true'
         id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
```

Comment:
This change is due to the old method being deprecated by GitHub: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/